build.gradle

```diff
@@ -132,11 +135,21 @@ sourceSets.main.java.srcDir "${openapiTargetDir}/src/main/java"
 ideaModule.dependsOn swaggerSources.server.code
 // end of OpenAPI Server Generation
 
+def googleCredentialsFile = "${projectDir}/src/test/resources/rendered/sa-account.json"
+
 test {
+    // Sets the application Google Credentials.
+    environment = [
+            'GOOGLE_APPLICATION_CREDENTIALS': "${googleCredentialsFile}"
+    ]
     useJUnitPlatform()
 }
 
 task unitTest(type: Test) {
+    // Sets the application Google Credentials.
+    environment = [
+            'GOOGLE_APPLICATION_CREDENTIALS': "${googleCredentialsFile}"
```

Comment:
why do our unitTests need google credentials? Shouldn't only integrationTets need the service account credentials?